x-pack/elastic-agent/docs/run-elastic-agent-standalone.asciidoc

```diff
@@ -28,12 +28,13 @@ See the https://www.elastic.co/downloads/elastic-agent[download page] for other
 installation options.
 --
 
-. Under **Choose an agent policy**, select a policy to use for the agent. The
-default policy includes a system integration for collecting logs and metrics
-from the host system.
+. Back in {fleet}, under **Choose an agent policy**, select a policy to use for
+the agent. The default policy includes a system integration for collecting logs
+and metrics from the host system.
 
 . Under **Configure the agent**, copy or download the policy. Copy this policy
-to the `elastic-agent.yml` on the host where the {agent} is extracted.
+to the `elastic-agent.yml` on the host where the {agent} is extracted. Not sure
+where to find the policy file? See <<installation-layout>>.
```

Comment:
I need to change this. The config file is not written to the directories described under installation-layout until _after_ the user runs the install command. So initially the user needs to add copy the configuration to the elastic-agent.yml where they've extracted the files. Then after installing the agent, I assume they'll need to go to the installed location.

I think this will be confusing for RPM/DEBIAN users where I assume the process is different because the install program isn't supported.